stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -336,7 +337,7 @@ internal class PaymentSheetViewModel internal constructor(
                 paymentIntent.value?.let {
                     resetViewState(
```

Comment:
you can make `resetViewState()` take a string res id

Reply:
In the scenario where there is an exception, I pass in the localized message, so it needs to be a string